move/operators/sources/operators/operators.move

```diff
@@ -0,0 +1,319 @@
+module operators::operators {
+    use sui::bag::{Self, Bag};
+
+    // -----
+    // Types
+    // -----
+
+    /// The `OwnerCap` capability representing the owner of the contract.
+    public struct OwnerCap has key, store {
+        id: UID,
+    }
+
+    /// The `OperatorCap` capability representing an approved operator.
+    public struct OperatorCap has key, store {
+        id: UID,
+    }
+
+    /// The main `Operators` struct storing the capabilities and operator IDs.
+    public struct Operators has key {
+        id: UID,
+        operator_ids: vector<ID>,
```

Comment:
```suggestion
        operators: VecSet<address>,
```
Use a `VecSet` and use `address` instead of ID since they're expected to be relayer addresses